sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -95,12 +95,12 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       e = intercept[AnalysisException] {
         sql(s"""LOAD DATA LOCAL INPATH "$testData" INTO TABLE $viewName""")
       }.getMessage
-      assert(e.contains(s"Target table in LOAD DATA cannot be temporary: `$viewName`"))
+      assert(e.contains(s"Target table in LOAD DATA does not exist: `$viewName`"))
```

Comment:
https://github.com/apache/spark/blob/c0ae6bc6ea38909730fad36e653d3c7ab0a84b44/sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala#L218-L223

Before this PR, `tableExists` checks the temp table, but `getTableMetadataOption` does not check it. Thus, instead of changing the test case, we need to change the impl of `LoadDataCommand`